units/en/unit3/agentic-rag/agent.mdx

```diff
@@ -0,0 +1,499 @@
+# Creating Your Gala Agent
+
```

Comment:
Is it possible to add a notebook version? Or people need to code it directly in their computer?

If we have a notebook it reduces the friction for our students. So if it's possible you can check other units to see how to add the button open in colab 🤗

Reply:
Since this is a more advanced unit, we thought of directly organizing it as a Python project instead of a Colab/notebook.